static/templates/stream_settings/stream_creation_form.hbs

```diff
@@ -3,36 +3,38 @@
     <form id="stream_creation_form">
         <div class="alert stream_create_info"></div>
         <div id="stream_creating_indicator"></div>
-        <div class="stream-creation-body">
-            <section class="block">
-                <label for="create_stream_name">
-                    {{t "Stream name" }}
-                </label>
-                <input type="text" name="stream_name" id="create_stream_name"
-                  placeholder="{{t 'Stream name' }}" value="" autocomplete="off" maxlength="{{ max_name_length }}" />
-                <div id="stream_name_error" class="stream_creation_error"></div>
-            </section>
-            <section class="block">
-                <label for="create_stream_description">
-                    {{t "Stream description" }}
-                </label>
-                <input type="text" name="stream_description" id="create_stream_description"
-                  placeholder="{{t 'Stream description' }}" value="" autocomplete="off" maxlength="{{ max_description_length }}" />
-            </section>
-            <section class="block" id="make-invite-only">
-                <div class="stream-types">
-                    {{> stream_types
-                      stream_post_policy=stream_post_policy_values.everyone.code
-                      is_stream_edit=false }}
-                </div>
-            </section>
-            <section class="block">
-                <label for="people_to_add">
-                    <h4 class="stream_setting_subsection_title">{{t "Choose subscribers" }}</h4>
-                </label>
-                <div id="stream_subscription_error" class="stream_creation_error"></div>
-                <div class="controls" id="people_to_add"></div>
-            </section>
+        <div id="create-stream-simplebar-wrapper" data-simplebar>
```

Comment:
I think we can avoid adding this extra element and instead add `data-simplebar` to `stream-creation-body` element itself.

Reply:
The problem with doing that was the scrollbar was overflowing when the number of users increased; So I decided to use an extra div which eliminated that problem. Sharing the videos below:

https://user-images.githubusercontent.com/59819879/160231538-e7d0bf85-512f-4af9-87ab-694bc469930e.mp4


https://user-images.githubusercontent.com/59819879/160231548-56a04434-afe9-458c-bbf8-58ef6fa6768d.mp4